### ISSUE TITLE
Plugin list in Settings Editor unreachable via Keyboard

### ISSUE THREAD

**Issue by IsabelParedes** [jupyterlab/jupyterlab#19027]
- Partially related to #15546

## Description

- The list of plugins on the left side of the settings editor is unreachable via keyboard.

## Reproduce

1. Go to Settings Editor
2. Use `Tab` or Arrow keys to attempt to navigate to any of the entries in the plugin list on the left side (`Cell Toolbar`, `Code Completion`, `Code Console`, etc.)

https://github.com/user-attachments/assets/ffaad30d-9a8a-4f99-993d-604d503387c6

## Expected behavior

- Users should be able to reach the plugin list when navigating with a keyboard (pressing `Tab` or `Shift` + `Tab`).
- The entries in the list should also be navigable with Up/Down arrow keys.
- Pressing `Enter`/`Space` when one of the entries is in focus should activate that tab.

## Context

- Operating System and version: Arch Linux
- Browser and version: Chromium v149.0.7827.53
- JupyterLab version: 4.5.8
